bee-network/src/config.rs

```diff
@@ -59,17 +62,18 @@ impl NetworkConfigBuilder {
             binding_addr: self.binding_addr.unwrap_or(crate::constants::DEFAULT_BINDING_ADDR),
             reconnect_interval: self
                 .reconnect_interval
-                .unwrap_or(crate::constants::DEFAULT_RECONNECT_INTERVAL),
+                .unwrap_or(crate::timing::DEFAULT_RECONNECT_INTERVAL),
         }
     }
 }
 
 /// Network configuration.
 #[derive(Clone, Copy, Debug)]
 pub struct NetworkConfig {
+    // TODO: use Port instead of u16
     pub(crate) binding_port: u16,
     pub(crate) binding_addr: IpAddr,
-    pub(crate) reconnect_interval: u64,
+    pub(crate) reconnect_interval: Seconds,
```

Comment:
Why not directly using a Duration btw ?

Reply:
Really good point. I don't know why I wanted something own. This is one of those situations where you're like: "What the hell was I thinking?" :joy: Maybe I though that `Seconds` would be better readable.